utilities/ride-model-util/src/test/java/com/adobe/ride/utilities/model/test/ModelObjectTest.java

```diff
@@ -206,6 +208,27 @@ public void testPresetsAndBuildNumberOfInstances() throws ModelSearchException {
     JSONArray objects = testObj.generateModelInstances(10);
     ModelObject.prettyPrintToConsole(objects);
   }
+  
+	@Test(suiteName = "smoke", groups = "integration", enabled = true)
+	public void testPrimitiveSchemaDefinitionModelInstance() {
+		final String schema = "{\"type\":\"integer\", \"format\":\"int32\"}";
+		final ModelObject modelObject = new ModelObject(schema, false);
+		final Object modelInstance = modelObject.buildValidModelInstance();
+		Assert.assertNotNull(modelInstance);
+	}
+
+	@Test(suiteName = "smoke", groups = "integration", enabled = true)
+	public void testFallBackToSchemaTypeIfFormatCannotBeIdentified() throws UnexpectedModelPropertyTypeException {
```

Comment:
Similar to the above test, I don't think I see code above to handle the fallback.  Do you have some code locally?

Reply:
PS method `getModelPropertyType()` lines 449-451.

Reply:
ah, missed it last time.  Comments added.